pkg/roachprod/roachprod.go

```diff
@@ -787,6 +848,8 @@ func Stop(ctx context.Context, l *logger.Logger, clusterName string, opts StopOp
 	if err != nil {
 		return err
 	}
+
+	_ = deleteClusterConfig(clusterName, l)
```

Comment:
It probably wouldn't hurt to log any errors here (and below) to make debugging easier in the future.